portfolio/src/main/webapp/script.js

```diff
@@ -150,6 +151,47 @@ function getComments()
     });
 }
 
+/** Check the login status to display the comments or force sign in */
+function checkLogin()
+{
+    var commentForm = document.getElementById('commentForm');
+    fetch('/home').then(response => response.json()).then((login) => {
+    if (login.loginStatus == 'true')
+    {
+        logButton(login.url, "Logout", 'logoutButton');
+        document.getElementById("loginButton").remove();
+    }
+    else (login.loginStatus == 'false')
+    {
+        hideComments();
+        logButton(login.url, "Login", 'loginButton');
+    }
+});
+}
+
+/** Create login/logout button depending on parameters */
+function logButton(urlText, name, id)
+{
+   var btn = document.createElement("BUTTON");
+    btn.innerHTML = name;
+    document.getElementById("content").appendChild(btn);  
+    btn.setAttribute('id', id);
+    var log = document.getElementById(id);
+    document.getElementById(id).onclick = function () {
+    location.href = urlText; 
+    };
+}
+
+/** Hide comment form and display message  */
+function hideComments()
+{
+    var allComments = document.getElementById('commentForm');
+    allComments.style.display = "none";
+    var message = document.createElement("H3");
+    message.innerText = "Sign In To See Comments";
+    document.getElementById("content").appendChild(message);
+}
+
```

Comment:
It is not related to this commit but rather to removeAllComments. 
There is one liner way to remove children. For example,
   commentObj.innerHTML = '';
or
  commentObj.textContent = '';

Reply:
This made it a lot simpler. Thank you!!